packages/shared/lib/core/governance/constants/proposal-status-poll-interval.constant.ts

```diff
@@ -0,0 +1,6 @@
+import { SECONDS_PER_MILESTONE } from '@core/network'
+import { MILLISECONDS_PER_SECOND } from '@core/utils'
```

Comment:
Add more specific imports

Reply:
We have to get rid of the barrel files to enforce this, otherwise DevExp is going to be downgraded by a lot :sweat_smile: